rust_src/src/multibyte.rs

```diff
@@ -583,6 +583,30 @@ pub fn multibyte_char_at(slice: &[c_uchar]) -> (Codepoint, usize) {
     }
 }
 
+/// Same as STRING_CHAR_AND_LENGHT
```

Comment:
```suggestion
/// Same as STRING_CHAR_AND_LENGTH
```
Typo

Reply:
Thought something looked off about the way I wrote that. Thanks.